<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PLSTestModule</class>
 <widget class="QWidget" name="PLSTestModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Auto Test Module</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>464</width>
        <height>702</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <item row="1" column="1">
          <widget class="QPushButton" name="initExceptionBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Go</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QPushButton" name="crashExceptionBtn">
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Go</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Lookup String</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QComboBox" name="initExceptionComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_16">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Crash exception</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLineEdit" name="lineEditLookupString"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_15">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Init exception</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QPushButton" name="pushButtonLookupString">
           <property name="text">
            <string>Go</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_17">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Runtime exception</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QComboBox" name="runtimeExceptionComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="runtimeExceptionBtn">
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Go</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QComboBox" name="crashExceptionComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="9" column="0" colspan="2">
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>masking interface test</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>10</number>
            </property>
            <item row="10" column="1">
             <widget class="QPushButton" name="pushButton_11">
              <property name="text">
               <string>Go</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLineEdit" name="lineEdit_5">
              <property name="placeholderText">
               <string>pls_masking_ip_info</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QPushButton" name="pushButton_7">
              <property name="text">
               <string>Go</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="pushButton_3">
              <property name="text">
               <string>Go</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QPushButton" name="pushButton_6">
              <property name="text">
               <string>Go</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string>Go</string>
              </property>
             </widget>
            </item>
            <item row="9" column="1">
             <widget class="QPushButton" name="pushButton_10">
              <property name="text">
               <string>Go</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLineEdit" name="lineEdit">
              <property name="placeholderText">
               <string>pls_masking_person_info</string>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QLineEdit" name="lineEdit_8">
              <property name="placeholderText">
               <string>pls_masking_Region_info</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLineEdit" name="lineEdit_2">
              <property name="placeholderText">
               <string>pls_masking_user_id_info</string>
              </property>
             </widget>
            </item>
            <item row="10" column="0">
             <widget class="QLineEdit" name="lineEdit_11">
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string>pls_masking_identify_info</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLineEdit" name="lineEdit_4">
              <property name="placeholderText">
               <string>pls_masking_address_info</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QPushButton" name="pushButton_5">
              <property name="text">
               <string>Go</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="pushButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Go</string>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <widget class="QPushButton" name="pushButton_8">
              <property name="text">
               <string>Go</string>
              </property>
             </widget>
            </item>
            <item row="8" column="1">
             <widget class="QPushButton" name="pushButton_9">
              <property name="text">
               <string>Go</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="pushButton_4">
              <property name="text">
               <string>Go</string>
              </property>
             </widget>
            </item>
            <item row="9" column="0">
             <widget class="QLineEdit" name="lineEdit_10">
              <property name="placeholderText">
               <string>pls_masking_datetime_info</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLineEdit" name="lineEdit_7">
              <property name="placeholderText">
               <string>pls_masking_bank_card_info</string>
              </property>
             </widget>
            </item>
            <item row="11" column="0">
             <widget class="QLineEdit" name="lineEdit_12">
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string>pls_masking_int_info</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLineEdit" name="lineEdit_3">
              <property name="placeholderText">
               <string>pls_masking_email_info</string>
              </property>
             </widget>
            </item>
            <item row="11" column="1">
             <widget class="QPushButton" name="pushButton_12">
              <property name="text">
               <string>Go</string>
              </property>
             </widget>
            </item>
            <item row="8" column="0">
             <widget class="QLineEdit" name="lineEdit_9">
              <property name="placeholderText">
               <string>pls_masking_passport_info</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLineEdit" name="lineEdit_6">
              <property name="placeholderText">
               <string>pls_masking_name_info</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLineEdit" name="lineEdit_13">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>pls_masking_double_info</string>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QPushButton" name="pushButton_13">
           <property name="text">
            <string>Go</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0" colspan="2">
          <widget class="QWidget" name="widget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="pushButtonQuit">
              <property name="text">
               <string>QApplication::Quit</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonExit">
              <property name="text">
               <string>c::exit</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonAbort">
              <property name="text">
               <string>c::abort</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonExitProcess">
              <property name="text">
               <string>win::ExitProcess</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
